pdf/plugin-polytracker/config.json5

```diff
@@ -0,0 +1,37 @@
+{
+  file_detail_views: {
+    polytracker: {
+      label: 'PolyTracker',
+      type: 'program_to_html',
+      exec: ['python3', 'main.py', '<inputFile>'],
```

Comment:
Need to add a `'<tempFile .db>'` which gets used as the polytracker db, rather than a hard-coded relative path in the script.

Reply:
Oh I actually have a local fix for that that I forgot to `git push`. Standby…